modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.tsx

```diff
@@ -60,6 +60,7 @@ const DropDown = ({
 				<ClayButton
 					aria-label={Liferay.Language.get('new')}
 					className="nav-btn nav-btn-monospaced"
+					data-qa-id="creationMenuNewButton"
```

Comment:
Could you make sure if this is a duplicate of another data-qa-id? If it is, please double check whether there is possibility of both uses of data-qa-id being on the same page.

Reply:
This is a duplicate of what you named it in CreationMenu.js. AFAIK you cannot have both implementations of CreationMenu on the same page.